kimchi/src/circuits/polynomials/range_check/mod.rs

```diff
@@ -0,0 +1,97 @@
+//! Range check gate
+
+///```text
+/// Range check field element structure:
+///
+///    Each field element a is decomposed into three 88-bit limbs a0, a1, a2 s.t. a = a0a1a2 in
```

Comment:
This is no longer true of this gate; all it's doing is 3 88-bit range checks. Can you rewrite this as `Every batch of 4 range-check gates performs 3 88-bit range checks` or equivalent?

Reply:
👍🏼  true story. I could not find a way to constraint the whole 264-bit element since it doesn't fit anywhere.  I will comment this.